helm/soperator-notifier/values.yaml

```diff
@@ -8,6 +8,9 @@ slack:
   # Whether to disable sending messages on changes of system job (health checks, etc.) statuses.
   ignoreSystemJobs: true
 
+  # Set true to ignore jobs without `user_mail` label`
+  forceUserMail: true
```

Comment:
Maybe `sendWithUserMailOnly` or something like that (`ignoreJobsWithoutUserMail`)? `force` looks very crutch to me

Reply:
Let's use `requireUserMail`